bin/build-helper

```diff
@@ -20,14 +20,12 @@ createImagesFileIfNotExists() {
 buildImage() {
   local image=$1
   createImagesFileIfNotExists
-  if [[ "$NODE_ENV" == "development" ]]; then
-    image_name=$(echo $image | cut -f1 -d":")
-    docker build -t ${REGISTRY}${image_name}:latest .
-    echo ${REGISTRY}${image_name}:latest >> $BASE_DIR/dist/images.txt
-  else
-    docker build -t ${REGISTRY}${image}$(getBranchSuffix)$(getCommitHash) .
-    echo ${REGISTRY}${image}$(getBranchSuffix)$(getCommitHash) >> $BASE_DIR/dist/images.txt
+  local tag=$(echo $image | cut -f2 -d":")
+  if [[ "$tag" != "latest" ]]; then
+    image=${REGISTRY}${image}$(getBranchSuffix)$(getCommitHash)
```

Comment:
Overriding a variable with itself is apparently a thing in bash.
```
$ foo=bar
$ foo=foo${foo}
$ echo $foo
foobar
```

Reply:
Could avoid mutation of `image` here by just making a function which returns what the value ought to be (something along these lines). I just think that mutation is yucky personally :P

```bash
getImageName() {
  local image = $1
  
  local tag=$(echo $image | cut -f2 -d":")
  
  if [[ "$tag" != "latest" ]]; then    
    return ${REGISTRY}${image}$(getBranchSuffix)$(getCommitHash)
  fi
  
  return image
```